### PR TITLE
created a merged document for building haiku

### PR DESCRIPTION
**Pull request by brjhaiku**

### CODE REVIEW

**Comment by pulkomandy** [comment]
Hi,
You are introducing yet another page about how to build Haiku. This leads to more repetition of the information, more difficult to follow website structure, etc.

I would prefer that you update an existing page instead, for example this one: https://www.haiku-os.org/documents/dev/building_haiku_ubuntu_linux_step_step/

This guide should be kept simple. So, I would skip the part about the non-x86 architectures, and just link to the already existing separate pages documenting them. The idea is that this page is the "main track", and cover the typical use case (you run Linux, you want to build x86 Haiku), and just provides references to other pages which explain what's different when you are running another operating system or want to build another architecture). This way, we provide an overview of the build process, and links to the relevant information for people who want to know more.

Also, I see part of your work is clean markdown, and part is quite messy HTML, which makes it hard to review the pull request. If at least the HTML was cleanly indented, it would be better.

**Comment by brjhaiku** [comment]
Hi
Thank you for the feedback. 

I've fixed all the issues mentioned.

**Comment by pulkomandy** [comment]
Thanks, looks good now. I will let other people review in case they have more comments :)

File: content/documents/dev/building_haiku_ubuntu_linux_step_step.html [hunk 1]
```diff
@@ -1,92 +1,82 @@
...
-<ol>
-<li>Install the needed packages for building:
+# Compiling for x86_64
+x86_64 Compiler Toolset
+=======================
 
-<pre class="terminal">sudo apt-get install subversion yasm autoconf automake texinfo flex bison gawk build-essential</pre></li>
+Building the x86_64 compiler toolset is quite easy and involves generating GCC
+binaries for your platform. For a complete list of flags for the configure
+script, see Haiku's [Configure Options]().
```
Comment (humdingerb) [inline content/documents/dev/building_haiku_ubuntu_linux_step_step.html]: The missing URL should be https://www.haiku-os.org/guides/building/configure , I suppose.

File: content/documents/dev/building_haiku_ubuntu_linux_step_step.html [hunk 2]
```diff
@@ -1,92 +1,82 @@
...
 
+To check whether your target architecture is support visit [Port Status](https://www.haiku-os.org/guides/building/port_status).
 
-    <div class="field field-type-text field-field-body">
-    <div class="field-items">
-            <div class="field-item odd">
-                    <div class="box-info">
-<strong>2010-January: This document is now obsolete. We are in the process of consolidating and re-organizing the website documentation. For now please refer to the <a href="/guides">in-progress website documentation</a>.</strong>
-</div>
+# Pre-requisite Software
```
Comment (humdingerb) [inline content/documents/dev/building_haiku_ubuntu_linux_step_step.html]: For the section headers you can decide to either use title casing ("Pre-requisite Software") or sentence casing ("Pre-requisite software"). Whatever you choose, stick with it for all headers in the document.

File: content/documents/dev/building_haiku_ubuntu_linux_step_step.html [hunk 3]
```diff
@@ -1,92 +1,82 @@
...
-<pre class="terminal">cd  buildtools/jam
-make
-sudo ./jam0 install
-cd ../..</pre></li>
+Compiling a nightly anyboot Haiku iso image
+--------------------------------------
 
-<li>Get the Haiku source (this will take a while):
+This is the standard build which which results in a live ISO that can be burned
+to an optical disc, or that can be written directly to a USB stick.
```
Comment (humdingerb) [inline content/documents/dev/building_haiku_ubuntu_linux_step_step.html]: Maybe shorten to "..., or written to a USB stick."

File: content/documents/dev/building_haiku_ubuntu_linux_step_step.html [hunk 4]
```diff
@@ -1,92 +1,82 @@
...
 
-For an existing Haiku developer:
 
-<pre class="terminal">svn checkout https://<span style="background-color: #fef389;">developername</span>@svn.berlios.de/svnroot/repos/haiku/haiku/trunk trunk</pre>
+Compiling a nightly raw disk images
+---------------------------------
 
-<p>If there are any errors when checking out above, just cd into the trunk directory and type "svn up" to get any missed files. This also applies when updating the code in the future.</p></li>
+This generates a simple raw disk image of Haiku which can be booted directly in
+a VM or written directly to a USB stick.
```
Comment (humdingerb) [inline content/documents/dev/building_haiku_ubuntu_linux_step_step.html]: No need for the second "directly" here.

**Comment by brjhaiku** [comment]
@humdingerb fixed all issues mentioned

File: content/documents/dev/building_haiku_ubuntu_linux_step_step.html (hunk 1 above)
Comment (brjhaiku) [reply content/documents/dev/building_haiku_ubuntu_linux_step_step.html]: @humdingerb I think I've added the missing URL. Please tell me if I should change anything else.

Thank you

Comment (humdingerb) [reply content/documents/dev/building_haiku_ubuntu_linux_step_step.html]: Thanks, looks good. Let's merge!

**Pull request merged**